### ISSUE TITLE
split the data wrt to the realization parameter

### ISSUE THREAD

**Issue by dufourc1** [dufourc1/ML_CS433_project2#14]
we should split the data for training and testing in the following way: for each event, we have multiple realization. So split 70% of these realizations (for example) for the training set and 30% for the test set. Do that for all the events and then you get the sets

**Comment by WilliamCappelletti** [comment]
It is done the function is train_test_split_realiz in module william
